### ISSUE TITLE
Exception with get_release method

### ISSUE THREAD

**Issue by karann-jain** [Microsoft/vsts-python-api#58]
I am getting below error while trying to get a release.

> Traceback (most recent call last):
>   File "/usr/local/lib/python3.5/dist-packages/msrest/serialization.py", line 1023, in __call__
>     value = self.deserialize_data(raw_value, attr_desc['type'])
>   File "/usr/local/lib/python3.5/dist-packages/msrest/serialization.py", line 1178, in deserialize_data
>     obj_type = self.dependencies[data_type]
> KeyError: 'number'
> 
> During handling of the above exception, another exception occurred:
> 
> Traceback (most recent call last):
>   File "/home/karan/PycharmProjects/vsts-automation/v2/sample.py", line 17, in <module>
>     release_ = client.get_release(project='myproject',release_id=123)
>   File "/usr/local/lib/python3.5/dist-packages/vsts/release/v4_1/release_client.py", line 628, in get_release
>     return self._deserialize('Release', response)
>   File "/usr/local/lib/python3.5/dist-packages/msrest/serialization.py", line 1023, in __call__
>     value = self.deserialize_data(raw_value, attr_desc['type'])
>   File "/usr/local/lib/python3.5/dist-packages/msrest/serialization.py", line 1176, in deserialize_data
>     return self.deserialize_type[iter_type](data, data_type[1:-1])
>   File "/usr/local/lib/python3.5/dist-packages/msrest/serialization.py", line 1203, in deserialize_iter
>     return [self.deserialize_data(a, iter_type) for a in attr]
>   File "/usr/local/lib/python3.5/dist-packages/msrest/serialization.py", line 1203, in <listcomp>
>     return [self.deserialize_data(a, iter_type) for a in attr]
>   File "/usr/local/lib/python3.5/dist-packages/msrest/serialization.py", line 1187, in deserialize_data
>     return self(obj_type, data)
>   File "/usr/local/lib/python3.5/dist-packages/msrest/serialization.py", line 1027, in __call__
>     raise_with_traceback(DeserializationError, msg, err)
>   File "/usr/local/lib/python3.5/dist-packages/msrest/exceptions.py", line 45, in raise_with_traceback
>     raise error.with_traceback(exc_traceback)
>   File "/usr/local/lib/python3.5/dist-packages/msrest/serialization.py", line 1023, in __call__
>     value = self.deserialize_data(raw_value, attr_desc['type'])
>   File "/usr/local/lib/python3.5/dist-packages/msrest/serialization.py", line 1178, in deserialize_data
>     obj_type = self.dependencies[data_type]
> msrest.exceptions.DeserializationError: Unable to deserialize to object: type, KeyError: 'number'

Below are the python package version i am trying to use.
pip3 list  | grep 'msrest\|vsts'
msrest                       0.4.28                
vsts                            0.1.3

**Comment by tedchamb** [comment]
@karann-jain Thanks for reporting this.  I am investigating.

**Comment by tedchamb** [comment]
@karann-jain As a workaround you may want to try and use the v4_0/release_client until this is fixed, assuming it does not have the same issue.

**Comment by karann-jain** [comment]
@tedchamb  v4_0/release_client also throws the same error.
I am using VSTS.
I noticed varying behavior with v4_1, for some releases i am getting valid response, but many are throwing this exception.

My Script Logic:

I have close to 30 release definitions and each having about 100 releases. I am running a script which loops through all release definitions and query in descending order to identify releases that i need.

sample test script.

```python
from vsts.vss_connection import VssConnection
from msrest.authentication import BasicAuthentication

token = 'my-token'
team_instance = 'vsts-base-url'

credentials = BasicAuthentication('', token)
connection = VssConnection(base_url=team_instance, creds=credentials)
client = connection.get_client('vsts.release.v4_0.release_client.ReleaseClient')

release_definitions = client.get_release_definitions(project='my-project')

for release_definition in release_definitions:
    releases=client.get_releases(project='my-project', definition_id=release_definition.id)
    for release in releases:
        release_ = client.get_release(project='my-project',release_id=release.id)
        print(release_.name)
```

**Comment by tedchamb** [comment]
Fixed in #60, and now available on pypi as version 0.1.4

**Comment by karann-jain** [comment]
Perfect works now. Thanks
